spring-kotlin/aws-ssh-server-manager/ci/bootstrap_test_db.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -e
+
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
+    CREATE USER myuser WITH LOGIN PASSWORD 'mypass';
+    GRANT USAGE ON SCHEMA public TO myuser;
+    GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA public TO myuser;
+    ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT, INSERT, UPDATE, DELETE ON TABLES TO myuser;
+    GRANT USAGE ON ALL SEQUENCES IN SCHEMA public TO myuser;
+    ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT USAGE ON SEQUENCES TO myuser;
+EOSQL
```

Comment:
Nit: missing new line ;)

Reply:
what's this file?

Reply:
Oh, you're one of those :) 

This file just bootstraps the database that is spun up by my `docker-compose.yaml`
A common pattern we use that it is handy to check results
It might be also handy to perform some integration/system testing (calling the API and check results) later on